static/app/views/performance/newTraceDetails/trace.tsx

```diff
@@ -220,6 +273,7 @@ function RenderRow(props: {
     return (
       <div
         className="TraceRow Autogrouped"
+        onClick={() => props.onRowClick(props.node)}
```

Comment:
I benchmarked this and havent noticed any impact, so no need to optimize event handlers just yet